### DISCUSSION TITLE
list of annoyings bugs before 1.0.5

### DISCUSSION THREAD

**Discussion by alaingdl** [gnudatalanguage/gdl#1790] [General]
Visible
* Maybe the most visible is #1701 (surface and !p.multi)

Annoying but not blocking
* #1787 (being always be able to compile without Eigen3)
* #1781 a _detail_ 
* #1706 not to forget
* #29 (and this bug is now in testsuite in the file `test_foreach.pro`
* #1697 (I started few clean up but it take time ...)

long term bugs or discrepancies : 
* shade_surf #1601
* #1734 because GAUSS2DFIT is widely used

Concerning OSX and MSwin, may be we will have to discuss again on a mechanism to copy
locally the .so/.dylib because on the computers I used for regression tests for OSX, even short term
stability of a previously compiled version is not guaranty, because of the frequent change in libs 
provided by Brew (version & path)

**Comment by GillesDuvert** [comment]
shade_surf reference above is wrong ?

**Comment by GillesDuvert** [comment]
Please update, many of these have been closed now.
